setup.cfg

```diff
@@ -26,7 +26,7 @@ package_dir=
     =src
 include_package_data = True
 install_requires =
-    aiohttp==3.9.1
+    aiohttp==3.9.2
```

Comment:
```suggestion
    aiohttp>=3.9.1,<4
```

Reply:
@efiop / @karajan1001, can you try committing above suggestion please? I tried with fork in https://github.com/karajan1001/aiooss2/pull/90, but I think I don't have proper credentials there.